lib/l10n/app_en.arb

```diff
@@ -0,0 +1,10 @@
+{
+    "initializing": "Initializing...",
+    "cont": "Continue",
+    "before_you_begin": "Before you begin",
+    "intro_text0": "Before you start using the Concordium Mobile Wallet, you have to set up a passcode and optionally biometrics.",
```

Comment:
Could we use more descriptive names instead of _0 and _1?

Reply:
Hmm, I'm not sure, I'm not sure there are more descriptive names, it's just one text split into two paragraphs.
I think the right thing is merging them? :thinking:

Reply:
Nitpicking: Is "cont" a common abreviation for "Continue" at concordium? Otherwise, I would just use the full name as the key for readability

Reply:
It is not a common abbreviation.
I did it because both `Continue` and `continue` are "bad" keys. `Continue` is rejected by the generator and `continue` causes compile issues because it is a keyword.
Maybe there is a better solution though? :thinking:

Reply:
PS: In my old job we were using string xml files, which is why we could get away with dots and we could use whatever keys we liked (and the reason we were using string xml files was that we had a huge language file already 5000+ words that was shared with other native apps)